src/fracTrackerMobileLayer.js

```diff
@@ -1,23 +1,101 @@
-fracTrackerMobileLayer = function(map) {
-  var FracTracker_mobile = L.esri.featureLayer({
-    url: 'https://services.arcgis.com/jDGuO8tYggdCCnUJ/arcgis/rest/services/FracTrackerMobileAppNPCAMesaVerdeNationalPark_051416/FeatureServer/0/',
-    simplifyFactor: 1,
-  });
-
-  FracTracker_mobile.bindPopup(function(layer) {
-    return L.Util.template('<p><strong>Id : </strong>{OBJECTID}<br><strong>FT_MV_ID : </strong>{FT_MV_ID}<br><strong>Long : </strong>{Long}<br><strong>Lat :</strong> {Lat} <br> <strong>Caption : </strong>{caption} <br> <strong>issue :</strong> {issue} <br> <strong>facility :</strong> {facility} <br><strong> Location :</strong> {location} <br> <strong>URL :</strong> <a href={URL2}>{URL2}</a> <br> <img src={URL2} height="280" width="290"></p>', layer.feature.properties);
-  });
-
-  FracTracker_mobile.on('loading', function(e) {
-    if (typeof map.spin === 'function') {
-      map.spin(true);
-    }
-  });
-  FracTracker_mobile.on('load', function(e) {
-    if (typeof map.spin === 'function') {
-      map.spin(false);
-    }
-  });
-
-  return FracTracker_mobile;
+L.GeoJSON.FractrackerMobileLayer = L.GeoJSON.extend(
+  {
+    options: { },
+
+    initialize: function(options) {
+      options = options || {};
+      L.Util.setOptions(this, options);
+      this._layers = {};
+    },
+
+    onAdd: function(map) {
+      map.on('moveend', this.requestData, this);
+      this._map = map;
+      this.requestData();
+    },
+
+    onRemove: function(map) {
+      map.off('moveend', this.requestData, this);
+      if (typeof map.spin === 'function') {
+        map.spin(false);
+      }
+      this.clearLayers();
+      this._layers = {};
+    },
+
+    requestData: function() {
+      var self = this;
+
+      (function() {
+        var bounds = self._map.getBounds();
+        var northEast = bounds.getNorthEast();
+        var southWest = bounds.getSouthWest();
+        var left = southWest.lng;
+        var right = northEast.lng;
+        var top = northEast.lat;
+        var bottom = southWest.lat;
+        var polygon = left + ' ' + top + ',' + right + ' ' + top + ',' + right + ' ' + bottom + ',' + left + ' ' + bottom + ',' + left + ' ' + top;
+        
+        var $ = window.jQuery;
+        var fractrackerMobile_url = 'https://cors-anywhere.herokuapp.com/https://api.fractracker.org/v1/data/report?page=1&results_per_page=250&q={"filters":[{"name":"geometry","op":"intersects","val":"SRID=4326;POLYGON((' + polygon +'))"}],"order_by":[{"field":"report_date","direction":"desc"},{"field":"id","direction":"desc"}]}';
```

Comment:
Hey,
What is this url cors-anywhere.herokuapp.com ?

Reply:
This is to remove the cors policy issue. Jeff had suggested using it in the issue?

Reply:
I know it doesn't fetch the results if this is not prepended. But I am planning to look into it more tomorrow.

Reply:
This allows us to fetch it despite CORS restrictions. We've confirmed this is OK with the FracTracker team. Thanks!